mvp/src/main/java/com/distillery/android/blueprints/mvp/architecture/BaseContractView.kt

```diff
@@ -0,0 +1,7 @@
+package com.distillery.android.blueprints.mvp.architecture
+
+interface BaseContractView {
```

Comment:
Minor: I think the word `Contract` adds not much context so it can be avoided
```suggestion
interface BaseView {
```